spec/controllers/user_deposits_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe UserDepositsController, type: :controller do
+  subject {response}
+
+  let(:token) {double acceptable?: true}
+  before {allow(controller).to receive(:doorkeeper_token) {token}}
+
+  describe "#create" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.